app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginViewModel.java

```diff
@@ -96,4 +98,19 @@ protected void onCleared() {
         super.onCleared();
         compositeDisposable.dispose();
     }
+
+    public void requestToken() {
+        RequestToken requestToken = new RequestToken();
+        requestToken.setEmail(login.getEmail());
+
+        compositeDisposable.add(loginModel.requestToken(requestToken)
```

Comment:
Instead of adding this, use
`.compose(dispose(compositeDisposable))`. We already have it defined in the `ViewTransformers` file.